spec/jobs/subscription_placement_job_spec.rb

```diff
@@ -147,8 +147,10 @@
           allow(job).to receive(:unavailable_stock_lines_for) { order.line_items }
         end
 
-        it "does not place the order, sends an empty_order email" do
+        it "does not place the order, clears, all adjustments, and sends an empty_order email" do
+          expect(order.adjustments).to_not be_empty
```

Comment:
I wonder if we really need to add this one. Our only expectation is the one you added below. What happens before is not that relevant IMO.

Reply:
Sure, I'm happy to remove it.